### DISCUSSION TITLE
PrivateGPT on windows 11 with 2 RTX3080 GPUs: only LLM chat works not QueryDocs

### DISCUSSION THREAD

**Discussion by maragem** [imartinez/privateGPT#1446] [Q&A]
Hello,

i have successfully installed privateGPT on Windows 11 with 2 RTX 3080 GPUs.  Driver Version: 546.33       CUDA Version: 12.3
the code runs well when using only the LLM chat. When i try to upload a document and use the QueryDocs it produced the following error:

  RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:1 and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)

can anyone help please?
thanks

**Comment by Luis-Sarabando** [comment]
There is no support for multiGPU as far as i know
